tests/e2e/files/playbooks/run_playbook_test_playbook.yml

```diff
@@ -0,0 +1,51 @@
+---
+- name: A playbook to run simulated post-processing tasks and emulate a first run failure
+  hosts: localhost
+  gather_facts: false
+  vars:
+    tempfile: "/tmp/eda-e2etest-tempfile"
+  tasks:
+    - name: Remediate a host
+      when: remediation_required
+      block:
+        - name: Check if temp file exists
+          ansible.builtin.stat:
```

Comment:
@ttuffin We have seen that for some reason the stat call on localhost causes the ansible-playbook to do an auto discovery of python. And the results are different on Mac and Linux.
We were able to fix this in PR #365 by adding the python interpreter in the inventory.yml

Reply:
Thanks @mkanoor, thats correct, we are checking for strings in stdout so it shouldn't be affected. Will keep it in mind though.